### DISCUSSION TITLE
<General Chat/>

### DISCUSSION THREAD

**Discussion by RoshanAnsy** [Connect-with-US/surprise_test#1] [General]
This is for General conversation......

**Comment by ShivamShroff** [comment]
okaay

**Comment by ShivamShroff** [comment]
MongoDB Atlas password = io2DXXaOwMWsd2BK
username = TeamY_123

**Comment by RoshanAnsy** [comment]
Check the pull request and accept .........
@ShivamShroff 
@Dataworld123

**Comment by RoshanAnsy** [comment]
Which API do we need to integrate?
 Have you figured that out yet?
@Dataworld123

**Comment by RoshanAnsy** [comment]
UI has been updated check the latest UI ...........

**Comment by RoshanAnsy** [comment]
Check out this https://www.conventionalcommits.org/en/v1.0.0/
for good commit message.

**Comment by RoshanAnsy** [comment]
Now that you will work on this repo, what changes do you want to make in it?
 All the conflicting code has been fixed.

**Comment by RoshanAnsy** [comment]
About :-> 1. about the projects, Aim of this project and how this can help you (  section 1)

               2. working diagram of our project including the all functionality with diagram (section 2)
               
Contact :-> 1.  Meet our team ( include social media )
                   2. Add redme.md file content

**Comment by ShivamShroff** [comment]
Project logo - ![QuizFuse](https://github.com/Connect-with-US/surprise_test/assets/106458765/0c6d10fe-7fbb-4c1e-8679-79176699fcc8)
Project Name - QuizFuse

**Comment by ShivamShroff** [comment]
The conflict has now been resolved, and everyone can resume work on the project.
